packages/lexical-clipboard/src/clipboard.ts

```diff
@@ -56,13 +56,15 @@ export function $getHtmlContent(editor: LexicalEditor): string | null {
     ($isRangeSelection(selection) && selection.isCollapsed()) ||
     selection.getNodes().length === 0
   ) {
-    return null;
+    return '';
   }
 
   return $generateHtmlFromNodes(editor, selection);
 }
 
-export function $getLexicalContent(editor: LexicalEditor): string | null {
+// TODO Return a blank string instead
+// TODO Rename to $getJSON
```

Comment:
Should this be done in a future feature version? 
```suggestion
// TODO 0.6. Rename to $getJSON
```